.github/workflows/test_and_deploy.yml

```diff
@@ -1,19 +1,18 @@
 # This workflows will upload a Python Package using Twine when a release is created
 # For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
-
 name: tests
 
 on:
   push:
     branches:
       - main
-      - main
+      - npe2
```

Comment:
I don't know why this change is in the cookiecutter, can someone confirm if it's ok to have here too?

Reply:
I don't think this is needed. It just means the tests will run when we push to a branch called "npe2". I presume it's there for plugins that need to transition from npe1 to npe2.